lib/tasks/example_data.rake

```diff
@@ -67,11 +67,28 @@ end
 
 def assessors
   [
-    { name: "Dave Assessor", email: "assessor-dave@example.com" },
-    { name: "Beryl Assessor", email: "assessor-beryl@example.com" },
+    {
+      name: "Dave Assessor",
+      email: "assessor-dave@example.com",
+      award_decline: true,
+    },
+    {
+      name: "Beryl Assessor",
+      email: "assessor-beryl@example.com",
+      award_decline: true,
+    },
   ]
 end
 
+def admin
+  {
+    name: "Sally Admin",
+    email: "admin-sally@example.com",
+    support_console_access: true,
+    award_decline: true,
```

Comment:
Might be good to have this set to false as once we implement the authorisation checks we'll want a user that doesn't have award_decline.